Airports/EGLF/Airspace.txt

```diff
@@ -1,76 +1,74 @@
-EGLF Farnborough ATZ N051.17.54.760 W000.50.13.800 N051.17.57.540 W000.49.43.780
-EGLF Farnborough ATZ N051.17.57.540 W000.49.43.780 N051.18.09.650 W000.47.34.100
-EGLF Farnborough ATZ N051.18.09.650 W000.47.34.100 N051.18.10.980 W000.47.21.160
-EGLF Farnborough ATZ N051.18.10.980 W000.47.21.160 N051.18.13.460 W000.47.11.020
-EGLF Farnborough ATZ N051.18.13.460 W000.47.11.020 N051.18.15.980 W000.47.03.980
-EGLF Farnborough ATZ N051.18.15.980 W000.47.03.980 N051.18.19.130 W000.46.55.640 
-EGLF Farnborough ATZ N051.18.19.130 W000.46.55.640 N051.18.32.610 W000.46.28.940 
-EGLF Farnborough ATZ N051.18.32.610 W000.46.28.940 N051.18.42.580 W000.46.06.980 
-EGLF Farnborough ATZ N051.18.42.580 W000.46.06.980 N051.18.48.110 W000.45.54.180 
-EGLF Farnborough ATZ N051.18.48.110 W000.45.54.180 N051.18.53.000 W000.45.42.420
-EGLF Farnborough ATZ N051.18.53.000 W000.45.42.420 N051.18.57.191 W000.45.29.371
\ No newline at end of file
+;Farnborough CTR1
+EGLF Farnborough CTA N051.21.12.000 W000.42.47.000 N051.21.03.000 W000.42.36.000 
+EGLF Farnborough CTA N051.21.03.000 W000.42.36.000 N051.20.35.000 W000.39.59.000 
+EGLF Farnborough CTA N051.20.35.000 W000.39.59.000 N051.15.20.000 W000.36.39.000 
+EGLF Farnborough CTA N051.15.20.000 W000.36.39.000 N051.10.35.000 W000.50.54.000 
+EGLF Farnborough CTA N051.10.35.000 W000.50.54.000 N051.17.05.000 W000.55.08.000 
+EGLF Farnborough CTA N051.17.05.000 W000.55.08.000 N051.21.12.000 W000.42.47.000
+;Farnborough CTR2
+EGLF Farnborough CTA N051.20.35.000 W000.39.59.000 N051.20.13.000 W000.38.00.000 
+EGLF Farnborough CTA N051.20.13.000 W000.38.00.000 N051.15.20.000 W000.36.39.000 
+EGLF Farnborough CTA N051.15.20.000 W000.36.39.000 N051.20.35.000 W000.39.59.000
+;Farnborough CTA1
+EGLF Farnborough CTA N051.20.13.001 W000.31.03.372 N051.17.41.000 W000.29.27.000 
+EGLF Farnborough CTA N051.17.41.000 W000.29.27.000 N051.15.20.000 W000.36.39.000 
+EGLF Farnborough CTA N051.15.20.000 W000.36.39.000 N051.20.13.000 W000.38.00.000 
+EGLF Farnborough CTA N051.20.13.000 W000.38.00.000 N051.20.13.001 W000.31.03.372
+;Farnborough CTA2
+EGLF Farnborough CTA N051.17.05.000 W000.55.08.000 N051.10.35.000 W000.50.54.000 
+EGLF Farnborough CTA N051.10.35.000 W000.50.54.000 N051.11.01.000 W000.53.55.000 
+EGLF Farnborough CTA N051.11.01.000 W000.53.55.000 N051.15.35.000 W000.57.48.000 
+EGLF Farnborough CTA N051.15.35.000 W000.57.48.000 N051.15.37.786 W000.57.44.872
+EGLF Farnborough CTA N051.15.37.786 W000.57.44.872 N051.15.57.308 W000.57.20.957
+EGLF Farnborough CTA N051.15.57.308 W000.57.20.957 N051.16.15.450 W000.56.54.416
+EGLF Farnborough CTA N051.16.15.450 W000.56.54.416 N051.16.32.075 W000.56.25.453
+EGLF Farnborough CTA N051.16.32.075 W000.56.25.453 N051.16.47.055 W000.55.54.286
+EGLF Farnborough CTA N051.16.47.055 W000.55.54.286 N051.17.00.276 W000.55.21.154
+EGLF Farnborough CTA N051.17.00.276 W000.55.21.154 N051.17.05.000 W000.55.08.000
+;Farnborough CTA3
```

Comment:
Could you format the comments as in the other areas 

;Farnborough CTA 1 Class D (4500-5000)